distribution/kernel/carbon-home/repository/resources/conf/default.json

```diff
@@ -222,5 +222,6 @@
   "tenant_mgt.enable_tenant_validation_for_nonsaas_username" : true,
   "jce_provider.provider_name" : "BC",
   "signature_util.enable_sha256_algo" : true,
-  "tenant_mgt.enable_input_validation" : false
+  "tenant_mgt.enable_input_validation" : false,
+  "tenant_mgt.user_info_handler" : ""
```

Comment:
having an empty value as the default feels like odd. Is there a better way to do this?

Reply:
Done